src/Compilers/VisualBasic/Portable/Symbols/TypeSymbolExtensions.vb

```diff
@@ -717,7 +717,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                     current.OriginalDefinition.AddUseSiteDiagnostics(useSiteDiagnostics)
                 End If
 
-                If current = superType Then
+                If current.IsSameType(superType, TypeCompareKind.IgnoreTupleNames) Then
```

Comment:
>IsSameType [](start = 27, length = 10)

It looks like IsDerivedFrom is using IsSameTypeIgnoringAll. It feels like they should be consistent. Perhaps should use IsSameTypeIgnoringAll here as well.